Dockerfile

```diff
@@ -1,20 +1,36 @@
-FROM node:8.9-alpine
+FROM node:10.10-alpine
 
-ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64 /usr/local/bin/dumb-init
+ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 /usr/local/bin/dumb-init
 RUN chmod +x /usr/local/bin/dumb-init
 
 RUN apk add --update --no-cache git curl
 
-RUN yarn global add npm
+ARG PORT=8080
+ARG NODE_ENV=production
+ARG GIT_COMMIT=unspecified
+ARG GIT_BRANCH=unspecified
+
+LABEL git_commit=$GIT_COMMIT
+LABEL git_branch=$GIT_BRANCH
 
-# Sets the HOME environment variable.
 ENV HOME=/home/app
 ENV PATH=/home/app/src/node_modules/.bin:$PATH
 ENV NPM_TOKEN=0
+ENV PORT $PORT
+ENV NODE_ENV $NODE_ENV
+ENV GIT_COMMIT $GIT_COMMIT
+ENV GIT_BRANCH $GIT_BRANCH
 
 RUN mkdir -p $HOME
 RUN echo '//registry.npmjs.org/:_authToken=${NPM_TOKEN}' > $HOME/.npmrc
 
+RUN npm i npm@latest -g
+
+COPY ./healthcheck.js /healthcheck.js
+HEALTHCHECK --interval=30s CMD node /healthcheck.js
```

Comment:
we might need to factor in route prefix for this

Reply:
lets turn off healthcheck by default and have it pass in url for each individual repo